src/backend/joanie/core/templates/issuers/contract_definition.html

```diff
@@ -0,0 +1,79 @@
+{% load i18n %}
+{% load static %}
+{% load howard_tags %}
+
+<html>
+  {% if debug %}
+  <head>
+    <style>
+      {{ css }}
+    </style>
+  </head>
+  {% endif %}
+  <body>
+    <div id="contract">
```

Comment:
Use `id` for css selector is a bad practice as the specificity of this selector is too high. `class` should be favored. In other project, we try to use [BEM](https://getbem.com/) naming convention for our CSS classes.